src/main/java/baritone/utils/PathRenderer.java

```diff
@@ -341,6 +341,17 @@ public static void drawDankLitGoalBox(Entity player, Goal goal, float partialTic
                 drawDankLitGoalBox(player, g, partialTicks, color);
             }
             return;
+        } else if (goal instanceof GoalYLevel) {
+            GoalYLevel goalpos = (GoalYLevel) goal;
+            minX = player.posX - Baritone.settings().ylevelboxsize.value - renderPosX;
+            minZ = player.posZ - Baritone.settings().ylevelboxsize.value - renderPosZ;
+            maxX = player.posX + Baritone.settings().ylevelboxsize.value - renderPosX;
+            maxZ = player.posZ + Baritone.settings().ylevelboxsize.value - renderPosZ;
+            minY = ((GoalYLevel) goal).getLevel() - renderPosY;
+            maxY = minY + 2;
+            double y = MathHelper.cos((float) (((float) ((System.nanoTime() / 100000L) % 20000L)) / 20000F * Math.PI * 2));
```

Comment:
Instead of copy pasting this hairy mess, move the entire `double y = MathHelper.cos((float) (((float) ((System.nanoTime() / 100000L) % 20000L)) / 20000F * Math.PI * 2));` before the if lol